.github/workflows/test-suite.yml

```diff
@@ -32,29 +35,14 @@ jobs:
             zsh -n "$file"
             echo "Success! 🎉"
           done
-      - name: Install Test - No Branch - Master
+      - name: Install Test - branch "${{ matrix.branch }}"
         shell: zsh {0}
         run: |
-          # Ensure Zap not already installed
-          rm -rf "${XDG_DATA_HOME:-$HOME/.local/share}/zap"
```

Comment:
Are we sure this is not required? As I have previously seen Zap installed on the first install step.  Initial thought was that it was due to GitHub caching though did not have the time to investigate.  IMO keeping this is neglible from a runtime cost.

Reply:
```zsh
[[ -d "${XDG_DATA_HOME:-$HOME/.local/share}/zap" ]] && rm -rf "${XDG_DATA_HOME:-$HOME/.local/share}/zap" || :
```